tests/utils/SparseGridTest.cpp

```diff
@@ -43,25 +43,38 @@ TEST_P(GetNearbyTest, GetNearby)
 
     const std::vector<typename SparsePointGridInclusive<Point>::Elem> result = grid.getNearby(target, grid_size);
 
-    //Are all near points reported as near?
+    // Are all near points reported as near?
     for (const Point point : parameters.expected_near)
     {
-        EXPECT_NE(result.end(), std::find_if(result.begin(), result.end(), [&point](const typename SparsePointGridInclusive<Point>::Elem &elem) { return elem.val == point; }))
-            << "Point " << point << " is near " << target << " (distance " << vSize(point - target) << "), but getNearby didn't find it. Grid size: " << grid_size;
+        EXPECT_NE(result.end(),
+                  std::find_if(result.begin(),
+                               result.end(),
+                               [&point](const typename SparsePointGridInclusive<Point>::Elem& elem) { return elem.val == point; }))
+          << "Point " << point << " is near " << target << " (distance " << vSize(point - target)
+          << "), but getNearby didn't find it. Grid size: " << grid_size;
```

Comment:
Indent with 4 spaces.

You should also break this line up in a better way, rather than halfway through the print of a coordinate. This can be made more readable.